README.md

```diff
@@ -45,17 +44,33 @@ access_token_secret: ACCESS TOKEN SECRET
 
 [twitter-app]: https://apps.twitter.com
 
+##### Location
+
+Currently `golden-hour` is hard-coded to use Seattle's location. To use weather and sunset times for a different location, you'll need to dig into the code a bit and change some things! To fix this, check out [#11](https://github.com/alanhussey/golden-hour/issues/11)
+
 ##### Dark Sky *(optional)*
 
 If you want the weather report, [get a Dark Sky API key][dark-sky-api].
 
 [dark-sky-api]: https://darksky.net/dev
 
+### Running
+
+Use the command-line help to check parameters:
+```
+~/golden-hour $ main.py --help
+usage: [...]
+```
+
 #### Running automatically
 
-Once you have everything set up, set up a cron job to run `golden-hour` at the same time every day. Make sure it runs at least one hour before the earliest sunset of the year.
+Once you have everything set up, set up a cron job to run `golden-hour` at the same time every day. Make sure it runs at least one hour before the earliest sunset of the year. An example crontab line may be:
+```
+0 15 * * * python3.6 ~pi/golden-hour/main.py --start-before-sunset 90 --post-to-twitter --darksky-key YOUR DARK SKY KEY
```

Comment:
Here's [my crontab and start script](https://gist.github.com/alanhussey/0f5ccbd1f28e1c7d2c851bff5c496889) btw. It might be valuable to link as an example?